Sources/SwiftCurrent_Testing/CustomAssertions.swift

```diff
@@ -18,6 +18,16 @@ import UIKit
 #if !os(watchOS)
 /// Assert that a workflow was launched and matches the workflow passed in
 public func XCTAssertWorkflowLaunched<F>(from VC: UIViewController, workflow: Workflow<F>, file: StaticString = #filePath, line: UInt = #line) {
```

Comment:
Perhaps this workflow should be optional, given the other version is?

Reply:
I don't think that would work here since it's generic on the parameter value. If you passed in nil it wouldn't know what the generic type should be. Plus it wouldn't be able to tell which overloaded function to call, since for nil the two signatures would be identical.

Reply:
I suppose another way to look at it would be to consider that this variant would be used for workflows created programatically. Do you envision a case where a programatic workflow might be optional?

Reply:
The only case I can dream up is if somebody either does `nil as Workflow<Type>` or has `let wf: Workflow<Type>?` and assigns it later. The compiler should disambiguate `nil` to the most reasonable candidate, which is `AnyWorkflow`.